code/modules/client/preference_setup/loadout/lists/accessories.dm

```diff
@@ -127,6 +127,11 @@
 	cost = 1
 	flags = GEAR_HAS_COLOR_SELECTION
 
+datum/gear/accessory/wristwatches
```

Comment:
```suggestion
/datum/gear/accessory/wristwatches
```

Reply:
> Checks aren't running since you're a first-time contributor but I doubt this would even compile because of this.

they should, I was able to compile the game and test it myself